docs/examples/pytorch/single_stage_detector/train.py

```diff
@@ -185,7 +186,7 @@ def train300_mlperf_coco(args):
         dboxes,
         args.seed)
     train_pipe.build()
-    train_loader = DALICOCOIterator(train_pipe, ["images", "boxes", "labels"], 118287)
+    train_loader = DALIGenericIterator(train_pipe, ["images", "boxes", "labels"], 118287)
```

Comment:
Please try to use `pipe.epoch_size("Reader")` instead of hard-coded `118287`.
And in the COCO pipeline itself please named reader `inputs, bboxes, labels = self.input(name="Reader")`

Reply:
Done